test/test-functions

```diff
@@ -6,7 +6,7 @@ export PATH
 
 LOOKS_LIKE_DEBIAN=$(source /etc/os-release && [[ "$ID" = "debian" || "$ID_LIKE" = "debian" ]] && echo yes || true)
 LOOKS_LIKE_ARCH=$(source /etc/os-release && [[ "$ID" = "arch" ]] && echo yes || true)
-LOOKS_LIKE_SUSE=$(source /etc/os-release && [[ "$ID_LIKE" = "suse" ]] && echo yes || true)
+LOOKS_LIKE_SUSE=$(source /etc/os-release && [[ "$ID_LIKE" =~ "suse" ]] && echo yes || true)
```

Comment:
I think maybe this would be more appropriate:

```
[[ " $ID_LIKE " = *" suse "* ]]
```

Since it would then match a full word.

Documentation of [ID_LIKE](https://www.freedesktop.org/software/systemd/man/os-release.html#ID_LIKE=) says "A space-separated list of operating system identifiers in the same syntax as the `ID=` setting."

Perhaps the other one above that looks at `$ID_LIKE` (for `debian`) should also be changed to do the same check as well, since it's conceivable that one might have multiple words too.

Cheers,
Filipe

Reply:
@keszybz The suggestion was adding spaces around the variable, as in `" $ID_LIKE "` so it matches if it's a single word or if the word is at the beginning or end...

Agree that it's unlikely to match other distros (now) but I find this is the kind of thing that when it causes a bug in the future, it's hard to trace and figure out why... Anyways, not blocking the substring match approach, but I still think matching whole words is still better.

Oh and in case you do full word matches, perhaps you'd want to match both `suse` and `opensuse` since both might appear in an `$ID_LIKE`.

Reply:
Oh, OK. Yeah, that would work and seems like a better approach indeed.